electrum/three_keys/script.py

```diff
@@ -100,11 +100,12 @@ def get_script_sig(self, signatures: List[str], public_keys: List[str]) -> str:
         if self._instant_recovery_alert_flag is None:
             raise ThreeKeysError('Recovery/alert/instant flag not set!')
         sigs = ''.join(push_script(sig) for sig in signatures)
+        self.get_redeem_script(public_keys)
```

Comment:
shouldn't it be deleted? we have `push_script(self.get_redeem_script(public_keys))` below

Reply:
Yes, will be done.